src/test/java/com/datadog/api/v1/client/api/DashboardsApiTest.java

```diff
@@ -520,35 +520,6 @@ public void dashboardLifecycleTest() throws ApiException {
         Widget timeseriesWidgetLogQuery = new Widget().definition(new WidgetDefinition(timeseriesWidgetDefinitionLogQuery));
         orderedWidgetList.add(timeseriesWidgetLogQuery);
 
-        // Timeseries Widget with Event query
-        TimeseriesWidgetDefinition timeseriesWidgetDefinitionEventQuery = new TimeseriesWidgetDefinition()
```

Comment:
Okay, so we're removing this because the timeseries widget will accept event queries in the form of `timeseriesWidgetLogQuery`